tests/cpp/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # Copyright (c) 2013-2015, EPFL/Blue Brain Project
 #                     Daniel Nachbaur <daniel.nachbaur@epfl.ch>
 #                     Raphael Dumusc <raphael.dumusc@epfl.ch>
+#
+# Change this number when adding tests to force a CMake run: 0
```

Comment:
That's correct, but I guess I just don't see this as a problem in practice. Jenkins does clean builds anyway. But fine with me, it's just a comment :-)